docs/service_spec.md

```diff
@@ -74,6 +80,9 @@ A `POST` request indicates that the test harness wants to start an instance of t
   * `bigSegments` (object, optional): Enables and configures Big Segments. Properties are:
     * `callbackUri` (string, required): The base URI for the big segments store callback fixture. See [Callback fixtures](#callback-fixtures).
     * `userCacheSize`, `userCacheTimeMs`, `statusPollIntervalMS`, `staleAfterMs`: These correspond to the standard optional configuration parameters for every SDK that supports Big Segments.
+  * `tags` (object, optional): If specified, this has options for metadata/tags (that is, values that are translated into an `X-LaunchDarkly-Tags` header):
+    * `applicationId` (string, optional): If present and non-null, the SDK should set the "application ID" property to this string.
```

Comment:
This is `applicationID` (capital D) in the implementatiion.

Reply:
Whoops - it's the implementation that is wrong. The JSON schema here is using JS conventions, not Go conventions.